### ISSUE TITLE
Error on server startup

### ISSUE THREAD

**Issue by T1MOXA** [robingrether/iDisguise#9]
```
[Server thread/ERROR]: Error occurred while enabling iDisguise v5.6.4-SNAPSHOT (Is it up to date?)
java.lang.NoClassDefFoundError: de/robingrether/util/StringUtil
	at de.robingrether.idisguise.management.VersionHelper.requireVersion(VersionHelper.java:32) ~[?:?]
	at de.robingrether.idisguise.management.VersionHelper.init(VersionHelper.java:70) ~[?:?]
	at de.robingrether.idisguise.iDisguise.onEnable(iDisguise.java:80) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:402) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:377) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:327) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:421) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:382) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:337) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:272) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:544) [spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
Caused by: java.lang.ClassNotFoundException: de.robingrether.util.StringUtil
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[?:1.8.0_111]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:101) ~[spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:86) ~[spigot-1.12.jar:git-Spigot-4df3c0c-03f1e37]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_111]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_111]
	... 14 more
```

**Comment by robingrether** [comment]
Where did you download that snapshot?

**Comment by luisagrether** [comment]
I'm so so so so so so sorry..... The download link was wrong and you did not download the whole plugin but only the core classes which are completely unable to run alone. Just go and download SNAPSHOT #4 once again. I fixed the link in the mean time https://www.robingrether.de/idisguise/

**Comment by T1MOXA** [comment]
There is a new problem.
Now is not working the right menu which is made using scoreboard

**Comment by luisagrether** [comment]
Scoreboard problems will be fixed in the next update.

**Comment by luisagrether** [comment]
Until the next update is released, you can use 5.6.4-SNAPSHOT #6 which you can grab over here: https://www.robingrether.de/idisguise/

**Comment by luisagrether** [comment]
Finally, all problems around scoreboard should be fixed in 5.6.4 snapshot #16 (or higher). You can download it over here: https://www.robingrether.de/idisguise/
